lib/oban/query.ex

```diff
@@ -70,11 +70,25 @@ defmodule Oban.Query do
   end
 
   @spec stage_scheduled_jobs(Config.t(), binary()) :: {integer(), nil}
-  def stage_scheduled_jobs(%Config{prefix: prefix, repo: repo, verbose: verbose}, queue) do
+  def stage_scheduled_jobs(%Config{} = config, queue) do
+    stage_scheduled_jobs(config, queue, fn queryable ->
+      where(queryable, [j], j.scheduled_at <= ^utc_now())
+    end)
+  end
+
+  @spec stage_all_scheduled_jobs(Config.t(), binary()) :: {integer(), nil}
+  def stage_all_scheduled_jobs(%Config{} = config, queue) do
+    stage_scheduled_jobs(config, queue, & &1)
+  end
+
+  @spec stage_scheduled_jobs(Config.t(), binary(), fun()) :: {integer(), nil}
+  def stage_scheduled_jobs(%Config{} = config, queue, filter) do
+    %{prefix: prefix, repo: repo, verbose: verbose} = config
```

Comment:
It seems redundant, but this needs to match on `%Config` for compile time key checking.